include/openssl/bio.h

```diff
@@ -272,6 +272,7 @@ typedef struct bio_method_st BIO_METHOD;
 
 const char *BIO_method_name(const BIO *b);
 int BIO_method_type(const BIO *b);
+int BIO_method_type_final(const BIO *b);
```

Comment:
This needs to go away (makes test cases fail)

Reply:
Fixed now, thanks